### PR TITLE
Use slack/wbw ids instead of names

### PR DESCRIPTION
**Pull request by pingiun**
This also changes the way the lowest wbw member is chosen, now a member
is randomly selected from all the members that have the common lowest
balance.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -284,55 +241,51 @@ def check(bot, remind=False):
...
+            f"{info['instr']}\n"
+            f"{lowest} has the honour to :bike: today"
+        )
+
+    except KeyError as e:
         bot.chat_post_message(
             channel, "Oh no something went wrong. "
                      "Back to the manual method, @pingiun handle this!"
         )
+        raise e
```
Comment (pingiun) [inline bot.py:288]: This will make sure that we receive a cron email to be able to debug the error

**Comment by joren485** [comment]
Did you test this?

**Comment by pingiun** [comment]
I tested that the slack mapping still works with my own user, but I did not check the random choosing when multiple people are tied for lowest balance

**Review by joren485 — APPROVED** [review]

**Comment by joren485** [comment]
Please run the Technicie role to pull the latest docker image when it is build.

**Pull request merged**